packages/suite/src/views/settings/coins/components/CoinsGroup/index.tsx

```diff
@@ -102,10 +103,20 @@ interface Props {
     unavailableCapabilities: { [key: string]: UnavailableCapability };
 }
 
-const Unavailable = ({ type }: { type: UnavailableCapability }) => {
+interface UnavailableMessageProps {
+    type: UnavailableCapability;
+    deviceVersion: number;
+    isBtcOnly: boolean;
+}
+const UnavailableMessage = ({ type, deviceVersion, isBtcOnly }: UnavailableMessageProps) => {
     switch (type) {
         case 'no-capability':
-            return <Translation id="FW_CAPABILITY_NO_CAPABILITY" />;
+            return deviceVersion === 1 && !isBtcOnly ? (
```

Comment:
im wondering if this case (xrp on T1) shoudn't be marked as "no-support"?
maybe it's a bug in connect

Reply:
No idea, but there was helpful explanation hidden in messages description
FW_CAPABILITY_NO_CAPABILITY: Firmware with missing capability (eg: LTC on Bitcoin-only FW, XRP on T1...)
FW_CAPABILITY_NO_SUPPORT: Similar to missing capability but tested on different level (coin info is missing in trezor-connect)